### PR TITLE
add chartmuseum 0.1.0 to incubator

### PR DESCRIPTION
**Pull request by jdolitsky**
Initial version of ChartMuseum chart using emptyDir volume per convo in #helm-users
https://github.com/chartmuseum/chartmuseum

### CODE REVIEW

File: incubator/chartmuseum/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: Helm Chart Repository with support for Amazon S3 and Google Cloud Storage
+name: chartmuseum
+version: 0.1.0
+home: https://github.com/chartmuseum/chartmuseum
+icon: https://raw.githubusercontent.com/chartmuseum/chartmuseum/master/logo.png
+maintainers:
+- name: ChartMuseum
+  email: chartmuseum@gmail.com
```
Comment (scottrigby) [inline incubator/chartmuseum/Chart.yaml]: Needs newline at EOF.

File: incubator/chartmuseum/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+            port: {{ .Values.service.internalPort }}
+        readinessProbe:
+          httpGet:
+            path: /index.yaml
+            port: {{ .Values.service.internalPort }}
+        volumeMounts:
+        - mountPath: /storage
+          name: storage-volume
+        resources:
+{{ toYaml .Values.resources | indent 10 }}
```
Comment (scottrigby) [inline incubator/chartmuseum/templates/deployment.yaml]: Needs newline at EOF.

File: incubator/chartmuseum/templates/service.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+{{ include "common.labels.standard" . | indent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+  - port: {{ .Values.service.externalPort }}
+    targetPort: {{ .Values.service.internalPort }}
+    protocol: TCP
+    name: {{ .Release.Name }}
+  selector:
+    app: {{ template "common.fullname" . }}
```
Comment (scottrigby) [inline incubator/chartmuseum/templates/service.yaml]: Needs newline at EOF.

File: incubator/chartmuseum/values.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+  type: ClusterIP
+  externalPort: 8080
+  internalPort: 8080
+resources:
+  limits:
+    cpu: 100m
+    memory: 128Mi
+  requests:
+    cpu: 80m
+    memory: 64Mi
```
Comment (scottrigby) [inline incubator/chartmuseum/values.yaml]: Needs newline at EOF.

**Review by scottrigby — COMMENTED** [review]
Preliminary syntax check. More substantive review later, or can tag-team with others

**Comment by nickadmolina** [comment]
CLA for nickadmolina now signed

File: incubator/chartmuseum/templates/deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  name: {{ include "common.fullname" . }}
+  labels:
+{{ include "common.labels.standard" . | indent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  template:
+    metadata:
+      name: {{ include "common.fullname" . }}
+      labels:
+{{ include "common.labels.standard" . | indent 8 }}
```
Comment (unguiculus) [inline incubator/chartmuseum/templates/deployment.yaml]: Only `app` and `release` label should go here.

File: incubator/chartmuseum/templates/service.yaml (hunk 3 above)
Comment (unguiculus) [inline incubator/chartmuseum/templates/service.yaml]: Selector should be:
```yaml
app: {{ template "common.name" . }}
release: {{ .Release.Name | quote }}
```

**Review by unguiculus — CHANGES_REQUESTED** [review]
Please fix the CLA.

File: incubator/chartmuseum/.gitignore [hunk 6]
```diff
@@ -0,0 +1,2 @@
+*.tgz
```
Comment (prydonius) [inline incubator/chartmuseum/.gitignore]: this file isn't necessary, we ignore this in the repo's root .gitignore

File: incubator/chartmuseum/requirements.yaml [hunk 7]
```diff
@@ -0,0 +1,4 @@
+dependencies:
```
Comment (prydonius) [inline incubator/chartmuseum/requirements.yaml:1]: yay!

**Review by jdolitsky — COMMENTED** [review]
@unguiculus @prydonius comments have been addressed

**Review by prydonius — APPROVED** [review]
this lgtm, will let @unguiculus @scottrigby take another look

**Comment by jdolitsky** [comment]
@prydonius thanks much. do you need us to squash commits?

**Comment by jdolitsky** [comment]
Jenkins failing with
`template: no template "common.fullname" associated with template "gotpl"`
Does the build process grab chart dependencies?

Should I commit charts/common-0.0.1.tgz?

File: incubator/chartmuseum/Chart.yaml [hunk 8]
```diff
@@ -0,0 +1,9 @@
+apiVersion: v1
+description: Helm Chart Repository with support for Amazon S3 and Google Cloud Storage
+name: chartmuseum
+version: 0.1.0
```
Comment (unguiculus) [inline incubator/chartmuseum/Chart.yaml:4]: Add `appVersion`

**Comment by prydonius** [comment]
@jdolitsky tests are looking good now, thanks again for the fix

**Comment by jdolitsky** [comment]
@unguiculus @scottrigby anything else?

**Review by unguiculus — CHANGES_REQUESTED** [review]

File: incubator/chartmuseum/Chart.yaml (hunk 8 above)
Comment (jdolitsky) [reply incubator/chartmuseum/Chart.yaml:4]: done.

**Review by unguiculus — APPROVED** [review]

**Pull request merged**